lib/kino/data_table.ex

```diff
@@ -113,12 +115,10 @@ defmodule Kino.DataTable do
   end
 
   defp infer_count({:columns, _, enum}, _) do
-    series = Enum.at(enum, 0)
-
-    case Enumerable.count(series) do
-      {:ok, count} -> count
-      _ -> nil
-    end
+    with {:ok, series} <- Enum.fetch(enum, 0),
```

Comment:
In practice this actually should never be the case, for `:rows` if the data is empty we don't know the columns, but for `:columns` if the data is empty we generally know the column names, it's just that they have no values. Doesn't hurt to use the safe access though.